externe_connecties/iv3_definitie_connectie.py

```diff
@@ -0,0 +1,72 @@
+from applicatie.logic.inlezen import ophalen_bestand_van_web
+from config.configurations import IV3_REPO_PATH
+import logging
+
+
+_logger = logging.getLogger(__file__)
+
+
+def geef_gebruikersvriendelijke_foutmeldingen(fouten):
```

Comment:
Goede functie. Een unit test voor functie `geef_gebruiksvriendelijke_foutmeldingen` lijkt me handig

Wel beetje tricky, want je wil in een unit test liever geen `fouten.json` ophalen uit de repo. Je mag een scheiding maken tussen logica en side effects en `fouten_bij_ophalen` meegeven als extra parameter.

Reply:
jep daar heb je gelijk in, ik kan denk ik wel een unittest hiervoor maken, ik mock dat ophalen van een bestand weg.

Reply:
unittest toegevoegd